competitions/template/etl/compose-and-upload

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017, 2019, 2020 Open Tech Strategies, LLC
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+__doc__ = """\
+Compose all of the Lever for Change <LONGCOMPETITIONNAME> Proposal CSV files.
```

Comment:
What is "LONGCOMPETITIONNAME"?  (Also, whatever it is, underscores please.)  Have we seen that variable before?  Internally we tend to use the short name everywhere.

Reply:
It's awesome to see this as a template file!  It would be good to document, in the comment at the top, what the substitution pattern is.  (When you process the template, you can replace that part of the comment with new text saying "This file was automatically generated from blah blah blah template; please make changes there not here" etc.)

Reply:
So, the long name is something like "Larsen/Lam ICONIQ Impact Award (2020)" which is different than "LLIIA2020"

Reply:
Gotcha.